cypress/support/pages/admin_console/Masthead.ts

```diff
@@ -7,19 +7,24 @@ export default class Masthead {
   private userDrpDwnKebab = "#user-dropdown-kebab";
   private isMobile = false;
 
-  isAdminConsole() {
+  checkIsAdminConsole() {
     cy.get(this.logoBtn).should("exist");
     cy.get(this.userDrpDwn).should("exist");
 
     return this;
   }
 
   get isMobileMode() {
+    if (window.parent[0].innerWidth >= 1024) {
+      this.isMobile = false;
+    } else {
+      this.isMobile = true;
+    }
+
     return this.isMobile;
   }
```

Comment:
If this is always the logic to determine if we are mobile or not we can get rid of the `isMobile` field entirely and return the result of the statement instead.

```suggestion
  get isMobileMode() {
    return window.parent[0].innerWidth >= 1024;
  }
```

Reply:
Updated